dll/win32/shell32/shlexec.cpp

```diff
@@ -1553,20 +1553,31 @@ static HRESULT ShellExecute_ContextMenuVerb(LPSHELLEXECUTEINFOW sei)
     if (FAILED_UNEXPECTEDLY(hr))
         return hr;
 
+    BOOL fDefault = !sei->lpVerb || !sei->lpVerb[0];
+
     CComHeapPtr<char> verb, parameters;
     __SHCloneStrWtoA(&verb, sei->lpVerb);
     __SHCloneStrWtoA(&parameters, sei->lpParameters);
 
     CMINVOKECOMMANDINFOEX ici = {};
     ici.cbSize = sizeof ici;
-    ici.fMask = (sei->fMask & (SEE_MASK_NO_CONSOLE | SEE_MASK_ASYNCOK | SEE_MASK_FLAG_NO_UI));
+    ici.fMask = (sei->fMask & (SEE_MASK_NO_CONSOLE | SEE_MASK_ASYNCOK | SEE_MASK_FLAG_NO_UI)) | CMIC_MASK_UNICODE;
```

Comment:
Does enabling unicode support as done there fix some extra stuff?

Reply:
Setting that flag means the caller should check lpParametersW instead and this is of course required if the parameter string contains something that cannot be converted to CP_ACP.

Reply:
ok yes I could imagine something like shortcut targets or program parameters specifying paths with non-latin-character names.